themes/bootprint3/theme.config.php

```diff
@@ -1,4 +1,142 @@
 <?php
 return [
-    'extends' => 'bootstrap3'
+    'extends' => 'bootstrap3',
+    'icons' => [
+        'sets' => [
+            /**
+             * Define icon sets here.
+             *
+             * All sets need:
+             * - 'template': which template the icon renders with
+             * - 'src': the location of the relevant resource (font, css, images)
+             * - 'prefix': prefix to place before each icon name for convenience
+             *             (ie. fa fa- for FontAwesome, default "")
+             */
+            'Fugue' => [
+                'template' => 'images',
+                'src' => 'icons', // Processed via imageLink
+            ],
+        ],
+        'aliases' => [
+            // UI
+            'arrow-right' => 'Fugue:arrow_right.png',
+            // 'barcode' => 'FontAwesome:barcode',
```

Comment:
Are these commented-out lines just for reference? Work in progress? Able to be removed?

Reply:
These are icons that I couldn't find a good Fugue replacement for. So they bleed through to Font Awesome.

Reply:
Makes sense... but the question is, is it better to leave them here as reminders/TODOs, or should we remove them on the assumption that we're unlikely to ever fill them in?

Reply:
I thought it was more transparent to show the gaps but I don't mind removing them either.